src/lib/datepicker/calendar.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('MatCalendar', () => {
         CalendarWithDateFilter,
       ],
       providers: [
-        MatDatepickerIntl,
+        CdkDatepickerIntl,
```

Comment:
I think we still want to keep `MatDatepickerIntl`, and make that extends `CdkDatepickerIntl`? For material datepicker users, it would be strange if sometimes they need the `Mat` version while other times they need the `Cdk` version.